.ado/templates/build-rnw.yml

```diff
@@ -57,6 +47,8 @@ steps:
       createLogFile: true
       logFileVerbosity: detailed
       msbuildArgs:
+        /restore
```

Comment:
Does this work on Desktop projects too?

Reply:
yes it should, msbuild added support for package.json.

I'm still struggling in this PR because it seems that msbuild uses different locations. So I'm aborting this change for now as the SDK"s put absolute paths in .appxrecipe and .dev.json files which have to match between the two and tests are failing...